examples/profile_scrape/instascrape_csv_example.py

```diff
@@ -0,0 +1,22 @@
+import json # import needed libraries / pip3 install json
+
+from instascrape import Profile # pip3 install insta-scrape
+
+import pandas as pd # pip3 install pandas
+
+google = Profile.from_username('google') # declare profile
+
+google.static_load() # scrape profile
+
+google_data = google.to_dict() # turn scraped data into python dictionary
+
+google_serialized = json.dumps(google_data) # serialize dictionary into json
+
+with open('google.json', 'w') as outfile: # write json data to "google.json"
+     json.dump(google_data, outfile) 
+
+with open('google.json', encoding='utf-8-sig') as f_input: # read json data from "google.json" and create pandas dataframe
+    df = pd.read_json(f_input)

```

Comment:
Could you reimplement this section to avoid using JSON? You can actually pass a dictionary right to a `pandas.DataFrame` and it'll build a DataFrame from the `dict`. Replacing it with something like this would work great:

```suggestion
google_data = {key: [val] for key, val in google_data.items()}   
df = pd.DataFrame(google_data)
```
This way we can avoid writing to JSON just to load it right back in, otherwise great example! This actually gives me an idea for a new [JSONScraper](https://github.com/chris-greening/instascrape/blob/master/instascrape/jsontools.py#L8) method I'm gonna implement lol